src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -203,7 +203,7 @@ public sealed class IidChangePointEstimator : TrivialEstimator<IidChangePointDet
         /// <p>Example code can be found by searching for <i>IidChangePointDetector</i> in <a href='https://github.com/dotnet/machinelearning'>ML.NET.</a></p>
         /// <format type="text/markdown">
```

Comment:
>/// <format type="text/markdown"> [](start = 8, length = 33)

please move the example in the XML comments of the class, line 190, rather than the constructor. 

Also the format node needs to be wrapped with the <example> node. PR #1558 is correcting this.